sdk/trace/src/main/java/io/opentelemetry/sdk/trace/data/SpanData.java

```diff
@@ -228,17 +215,6 @@ static Link create(SpanContext spanContext, Attributes attributes, int totalAttr
     /** Returns the {@code SpanContext} of the span this {@link Link} refers to. */
     SpanContext getSpanContext();
 
-    /**
-     * Returns the {@code SpanContext}.
-     *
-     * @return the {@code SpanContext}.
-     * @deprecated Use {@link #getSpanContext()}
-     */
-    @Deprecated
-    default SpanContext getContext() {
```

Comment:
Should we deprecate the other accessors for ids in favor of this one?

It has already helpers for bytes for ids and we can remove the helpers to convert.

Reply:
This one was deprecated in favor of `getSpanContext()`, so I think we want to delete this one, either way.

Reply:
Good point :)

Reply:
This is in the link not in the SpanData :)